doc/nrf/protocols/matter/end_product/configuring_cd.rst

```diff
@@ -0,0 +1,78 @@
+.. _ug_matter_device_configuring_cd:
+
+Storing Certification Declaration in firmware
+#############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Certification Declaration (CD) is :ref:`one of the documents <ug_matter_device_certification_results>` obtained as a result of a successful Matter certification.
+It is |matter_cd_definition|.
+It is used for this purpose in the :ref:`ug_matter_device_attestation` procedure during commissioning, where it is provided to a device as an element of :ref:`Device Attestation data <ug_matter_device_attestation_device_data>`.
+
+CD can only be stored in parts of firmware that can be modified during the device lifetime.
+With each new software version, the device manufacturer must apply for :ref:`ug_matter_device_certification` using one of the accepted paths.
+With an update to the device firmware, the new CD obtained as a result of the new certification process must replace the existing CD on the device.
+
+The following sections describe where and how you can place CD in firmware and how you can test it before applying for certification.
+
+Storing Certification Declaration
+*********************************
+
+In the |NCS| implementation of Matter, you can configure CD by adding the :c:macro:`CHIP_DEVICE_CONFIG_CERTIFICATION_DECLARATION` define in the :file:`chip_project_config.h` file.
+You can locate the array of bytes related to CD by running the search with the following condition:
+
+.. code-block:: console
+
+   cat CD.der | xxd -I
+
+Storing Certification Declaration in Zephyr
+===========================================
+
+Alternatively, you can opt for storing CD in Zephyr's Settings subsystem, which allows for storing data even after the device has been programmed.
+For example, this lets you add CD to the subsystem through CLI, or use Zephyr's API to store CD in the Settings subsystem within the code.
+
+To enable this configuration method, set the :kconfig:option:`CONFIG_CHIP_CERTIFICATION_DECLARATION_STORAGE` Kconfig option in the :file:`prj.conf` file instead of the define in :file:`chip_project_config.h`.
+
+.. _ug_matter_device_configuring_cd_generating_steps:
+
+Generating Certification Declaration for integration testing
+************************************************************
+
+To generate CD for integration testing, complete the following steps:
+
+1. Install the chip-cert tool.
+   This is a Matter tool used for generating new certificates.
+   Complete the following steps:
+
+   a. Navigate to the :file:`connectedhomeip` root directory.
+   a. In a terminal, run the following command to build the tool executable file:
+
+      .. code-block:: console
+
+         cd src/tools/chip-cert && gn gen out && ninja -C out chip-cert
+
+1. Run the following command pattern to generate CD:
```

Comment:
```suggestion
2. Run the following command pattern to generate CD:
```

Reply:
That is actually OK for RST and Markdown, but I changed it.